<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WorkspaceDevProcessDialog</class>
 <widget class="QDialog" name="WorkspaceDevProcessDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>700</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Messages</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2" stretch="1,0">
      <item>
       <widget class="QPlainTextEdit" name="MessagesTextEdit">
        <property name="undoRedoEnabled">
         <bool>false</bool>
        </property>
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="plainText">
         <string notr="true"/>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="CopyClipboardButton">
          <property name="text">
           <string>Copy to clipboard</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="SaveButton">
          <property name="text">
           <string>Save as...</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QProgressBar" name="ProgressBar">
     <property name="value">
      <number>24</number>
     </property>
     <property name="format">
      <string>%v/%m</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="ButtonBox">
     <property name="standardButtons">
      <set>QDialogButtonBox::Close</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
